src/channel.c

```diff
@@ -5149,9 +5149,7 @@ get_channel_arg(typval_T *tv, int check_open, int reading, ch_part_T part)
 	    channel = tv->vval.v_job->jv_channel;
     }
     else if (tv->v_type == VAR_CHANNEL)
-    {
```

Comment:
No need to remove.

Reply:
Fixed two places.  Thank you for reviewing.